bin/publish-modules

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env node
+
+var exec = require( 'child-process-promise' ).exec;
+var semver = require( 'semver' );
+
+console.log( 'Modules requiring publish:' );
+
+function getAllModules() {
+	var command = 'find client -type f -print | grep \'package.json\'';
+	return exec( command ).then( function( result ) {
+		return result.stdout.split( /\s+/ );
```

Comment:
What if a directory has a space in it? In practice very unlikely, but still covering our bases makes sense.